unittests/test-myfs.cpp

```diff
@@ -283,11 +288,17 @@ TEST_CASE( "read/write IMap", "[imap]" ) {
         imap->init();
         //mark the first entry as occupied for later comparison purposes
         imap->occupyIMapEntry(0);
+        REQUIRE(imap->getNextFreeInode(0)==1);
         imap->write(&bd);
         //occupy all imap entries and validate that
         for(int i = 0; i < NUM_DIR_ENTRIES; i++){
             imap->occupyIMapEntry(i);
             REQUIRE(imap->getIMapEntry(i)==true);
+            if(i==NUM_DIR_ENTRIES-1){
```

Comment:
hat nichts innerhalb der for schleife zu suchen, sollte unter der schleife platziert werden, evtl muss für den else branch eine iteration entfallen